benches/benches/db_lookup_times_utils/matrix.rs

```diff
@@ -0,0 +1,13 @@
+pub const BLOCK_COUNT_MATRIX: [u32; 2] = [10, 100];
+pub const TX_COUNT_MATRIX: [u32; 2] = [100, 1000];
+
+// todo: we can make this lazy loaded
+pub fn matrix() -> Box<dyn Iterator<Item = (u32, u32)>> {
```

Comment:
What is the reason for boxing the iterator here instead of just returning `impl Iterator`?

Reply:
addressed in c619b937b